Prox/Prox/Data/PlacesController.swift

```diff
@@ -34,6 +35,11 @@ class PlacesController {
         return manager
     }()
 
+    private lazy var radius: Double = {
+        let key = RemoteConfigKeys.searchRadiusInKm
+        return FIRRemoteConfig.remoteConfig()[key].numberValue!.doubleValue
```

Comment:
Yes. There is a value in the `RemoteConfigDefaults.plist`.

The only thing to worry about here: what happens if we fat finger a setting
in the RemoteConfig console, such that we don't have a valid number here.

Mitigating this would mean moving the defaults out of the plist and into
code, and about a screenful of more code.

e.g.

```swift
class RemoteConfigKeys {
    public let searchRadiusInKm =
RemoteConfigValueDouble(key:"search_radius_in_km", defaultValue: 4)
}
```

Then when we use it:

```swift
let radius = RemoteConfigKeys.searchRadiusInKm.value
```

This would be nicer, but I wanted to land a simple thing before landing the
more engineered version.